config_system/tests/depends_on_compare.test

```diff
@@ -0,0 +1,38 @@
+# This test checks the case where config dependency is defined by option of other type than configuration one
```

Comment:
What do you mean by "option of other type than configuration one"?

Maybe a better description would be: "This test checks 'depends on' statements using operators other than '&&' and '||'"?

The issue was that the operators weren't supported, not that the dependency was on a config option with a different type.

Reply:
Sure